src/shared/components/results/DidYouMean.tsx

```diff
@@ -0,0 +1,177 @@
+      >
+        {query}
+      </Link>,
+    ])}
+    {` in ${searchableNamespaceLabels[namespace]}`}
+  </li>
+);
+
+const didYouMeanNamespaces: SearchableNamespace[] = [
+  Namespace.uniprotkb,
+  Namespace.uniref,
+  Namespace.uniparc,
+  Namespace.proteomes,
+];
+
+type NamespaceSuggestions = {
+  namespace: SearchableNamespace;
+  suggestions: Suggestion[];
+};
+
+const TIMEOUT = 3_000;
+
+type DidYouMeanProps = {
+  suggestions?: Suggestion[];
+};
+
+const DidYouMean = ({ suggestions }: DidYouMeanProps) => {
+  const currentNamespace = useNS();
+  const location = useLocation();
+  const [dataLoading, setDataLoading] = useSafeState(true);
+  const [hasTimedOut, setHasTimedOut] = useSafeState(false);
+  const [otherNamespaceSuggestions, setOtherNamespaceSuggestions] =
+    useSafeState<NamespaceSuggestions[]>([]);
+
+  const { query } = parseQueryString(location.search);
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      setHasTimedOut(true);
+    }, TIMEOUT);
+    return () => clearTimeout(timeout);
+  }, [setHasTimedOut]);
+
+  useEffect(() => {
+    if (!query || !currentNamespace) {
+      return;
+    }
+
+    const otherNamespaces = didYouMeanNamespaces.filter(
+      (ns) => ns !== currentNamespace
+    );
+
+    const promises = otherNamespaces.map((ns) =>
+      fetchData<{
+        results: APIModel[];
+      }>(
+        queryString.stringifyUrl({
+          url: apiUrls.search(ns),
+          query: { query, size: 0 },
+        })
+      )
+    );
+
+    Promise.all(promises).then((responses) => {
+      const suggestionsWithData: NamespaceSuggestions[] = [];
+      for (const [namespace, response] of zip(otherNamespaces, responses)) {
+        /* istanbul ignore if */
+        if (!namespace) {
+          break; // Shouldn't happen, used to restrict types
+        }
+        const hits = +(response?.headers?.['x-total-records'] || 0);
+        if (hits) {
+          suggestionsWithData.push({
+            namespace,
+            suggestions: [{ query, hits }],
+          });
+        }
+      }
+      setOtherNamespaceSuggestions(
+        orderBy(suggestionsWithData, ({ suggestions }) => suggestions[0].hits, [
+          'desc',
+        ])
+      );
+      setDataLoading(false);
+    });
+  }, [currentNamespace, query, setDataLoading, setOtherNamespaceSuggestions]);
+
+  if (dataLoading && !hasTimedOut) {
+    return <Loader />;
+  }
+
+  const suggestionsSortedByHits = orderBy(suggestions, ['hits'], ['desc']);
+  const querySuggestions: NamespaceSuggestions[] = [
+    ...(suggestionsSortedByHits.length && currentNamespace
+      ? [
+          {
+            suggestions: suggestionsSortedByHits,
+            namespace: currentNamespace as SearchableNamespace,
+          },
+        ]
+      : []),
+    ...otherNamespaceSuggestions,
+  ];
+
+  return (
+    <Message level="info" className={styles['did-you-mean-message']}>
+      <h4>Sorry, no results were found!</h4>
```

Comment:
This is the only heading on this page, so it should be a h1, with a classname of `small` if we want to keep the size of a h4